analysis_driver/report_generation/run_crawler.py

```diff
@@ -3,7 +3,7 @@
 from collections import defaultdict, Counter
 from egcg_core import util
 from egcg_core.constants import *
-from egcg_core.rest_communication import post_or_patch as pp
+from egcg_core.rest_communication import patch_entry, post_or_patch as pp
```

Comment:
Should we be using patch? I have never used it in the past personally, and I've ran a quick search and have found the IETF guideline which states that patch is not safe to use. 
https://tools.ietf.org/html/rfc5789

Reply:
The word safe in this document just means that the method can change existing data which is the point of PATCH and PUT.
See https://stackoverflow.com/questions/41390997/why-patch-is-neither-safe-nor-idempotent
for a better explanation. Also note that Eve uses etags to ensure you're updating the latest version of the resource.

Reply:
@tcezard What do you mean by Eve uses etags, not sure I understand?

Reply:
See http://docs.python-eve.org/en/latest/features.html?highlight=etag#data-integrity-and-concurrency-control

Reply:
Thanks @tcezard